antarest/study/business/areas/renewable_management.py

```diff
@@ -388,6 +387,7 @@ def update_renewables_props(
         for area_id, update_renewables_by_ids in update_renewables_by_areas.items():
             old_renewables_by_ids = old_renewables_by_areas[area_id]
             for renewable_id, update_cluster in update_renewables_by_ids.items():
+                renewable_id = renewable_id.lower()
```

Comment:
In the same idea as other comments:
this should probaby be ensured upfront, in the inputs of that method we should have validated models (for ex. we have RenewableClusterInput objects).

Plus, it's not clear why we just transform it to lower case and not to ID.